package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@creativecommons/cc-assets",
+  "version": "0.1.0",
+  "description": "Assets used across Creative Commons projects.",
+  "main": "",
+  "scripts": {},
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/creativecommons/cc-assets.git"
+  },
+  "keywords": [
+    "PNG",
+    "SVG"
+  ],
+  "author": "Creative Commons",
+  "license": "SEE LICENSE IN LICENSE",
```

Comment:
Note: a `license` value is required in order to publish this package publicly. Hence, we refer to the LICENSE text file for details of the Copyright license pertaining to these assets.

Reply:
I mention the above because I cannot find any copyright terms for the CC assets. I've only found information on our trademark restrictions so far.